pkg/migrations/sql/preinstall/013-telemetry.sql

```diff
@@ -0,0 +1,22 @@
+CREATE TABLE IF NOT EXISTS SCHEMA_PS_CATALOG.promscale_instance_information (
+    uuid                                            UUID NOT NULL PRIMARY KEY,
+    last_updated                                    TIMESTAMPTZ NOT NULL,
+    telemetry_metrics_ingested_samples              BIGINT NOT NULL,
+    telemetry_metrics_queries_executed              BIGINT NOT NULL,
+    telemetry_metrics_queries_timed_out             BIGINT NOT NULL,
+    telemetry_metrics_queries_failed                BIGINT NOT NULL,
+    telemetry_traces_queries_executed               BIGINT NOT NULL,
+    telemetry_traces_dependency_queries_executed    BIGINT NOT NULL,
+    deletable                                       BOOLEAN DEFAULT TRUE
+);
+GRANT SELECT ON TABLE SCHEMA_PS_CATALOG.promscale_instance_information TO prom_reader;
+GRANT SELECT, INSERT, UPDATE, DELETE ON TABLE SCHEMA_PS_CATALOG.promscale_instance_information TO prom_writer;
+
+-- Write a counter reset row, i.e., the first row in the table. Purpose:
+-- The above telemetry_.* rows logically behave as counter. They get deleted by
+-- telemetry-housekeeper promscale when last_updated is too old to be stale. Since
```

Comment:
I don't quite understand this. Where is the code for the telemetry-housekeeper? I want to try to figure it out.

Reply:
Sorry, this wasn't supposed to be part of this PR. Its basically part of the big PR ([here](https://github.com/timescale/promscale/pull/884)), which I broke down into smaller PRs, this being the first in series. Will remove this part. Thank you.